controllers/flowlogspipeline/flp_objects.go

```diff
@@ -288,6 +352,7 @@ func (b *builder) buildPipelineConfig() ([]config.Stage, []config.StageParam) {
 		})
 	} else {
 		b.addTransformStages(&pipeline)
+		b.addMetricsStages(pipeline)
```

Comment:
maybe the name should be 'merge metrics configuration`

Reply:
OK